dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -147,3 +147,18 @@ error.invalid.packaged.liberty.jar.useraction=Configure a valid packaged Liberty
 error.application.not.found.inside.packaged.liberty.jar=CWWKC0266E: The application failed to start because the Spring Boot application was not found inside the packaged Liberty Server JAR file.
 error.application.not.found.inside.packaged.liberty.jar.explanation=The packaged Liberty Server JAR file must contain a configured Spring Boot application.
 error.application.not.found.inside.packaged.liberty.jar.useraction=Configure a valid packaged Liberty server JAR file.
+
+#ADDED for SpringBoot 4.0:
+error.spring4.required=CWWKC0276E: The {0} feature must be configured in the server.xml file. The {1} feature is configured and the application has Spring {2} content.
+error.spring4.required.explanation=The currently configured Spring Boot feature is not compatible with application content.
+error.spring4.required.useraction=Configure a version of the Spring Boot feature that is compatible with application content.
+
+#ADDED for SpringBoot 4.0:
+error.spring4.requires.servlet61.application=CWWKC0277E: The servlet-6.1 feature must be configured. The springBoot-4.0 feature is configured and the application uses servlets.
+error.spring4.requires.servlet61.application.explanation=The application uses Servlets but a no Servlet feature is configured.
+error.spring4.requires.servlet61.application.useraction=Configure the specified Servlet feature.
+
+#ADDED for SpringBoot 4.0:
+error.spring4.requires.websocket2.application=CWWKC0278E: Feature websocket-2.0 must be configured: Feature springBoot-4.0 is configured and the application uses Web Sockets.
```

Comment:
Please use the "The feature-name feature" format. Also, there isn't a "Web Socket" feature, but there is a "WebSocket" feature.
> The websocket-2.0 feature must be configured: The springBoot-4.0 feature is configured and the application uses WebSocket.